package.json

```diff
@@ -46,6 +46,7 @@
     "test:unit": "jest src",
     "test:bundlewatch": "bundlewatch --config .bundlewatch.config.js",
     "test:broken-links": "node ./scripts/test-filenames.js",
+    "test:sdk": "jest src/sdk",
```

Comment:
This is a duplicate.
The script `test:unit` already covers that.
So running `npm run test` will run SDK tests twice.
If you just need to run SDK test for development, you can run `npx jest src/sdk`